lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1201,40 +1219,45 @@ function ( $key ) {
 	private static function remove_insecure_settings( $input ) {
```

Comment:
@jorgefilipecosta I'd like to gather your thoughts on what's the expected input & output for `remove_insecure_properties` (as [in use here](https://github.com/WordPress/gutenberg/blob/trunk/lib/global-styles.php#L244)).

My understanding is that the input will always be user data. Then, if we are only expecting to process user data, presets should come unkeyed, and after the method runs the output should also be unkeyed. Do you have thoughts on this? Are there any other use cases I'm missing?

Reply:
At https://github.com/WordPress/gutenberg/pull/32622/commits/f84c4cb539e098147f656c69a1d30981cdb3923a I've refactored `removed_insecure_properties` so it's origin agnostic (and also static).

Reply:
Yes, it should only be used to process user data, theme data should be able to contain anything including "unsafe" properties. The changes seem good 👍